components/shopify_partner/queries/getAppInstalls.js

```diff
@@ -0,0 +1,45 @@
+const { gql } = require("graphql-request");
```

Comment:
Now that you have the components working, can you try to convert to ESM? If you encounter the same errors you saw before, we should figure out what the blocker is in the API and address that.